src/store/suggestion/reducer.js

```diff
@@ -0,0 +1,22 @@
+const initialState = {
+	suggestion: null,
+	data: [],
+	uploadImage: null,
+}
+
+export default (state = initialState, action) => {
+	switch (action.type) {
+		case 'SUGGESTIONS_FETCHED':
+			return {
+				suggestion: action.payload.data.suggestion,
+				data: action.payload.data.data,
+				uploadImage: action.payload.image,
```

Comment:
not necessarily meaning that you need to do it here, but good example to remind you that nested destructuring is also a thing:
```
const { image, data: { data, suggestion } } = action.payload
```